frontend/src/scenes/notebooks/Notebook/migrations/migrate.ts

```diff
@@ -82,11 +83,19 @@ function convertInsightQueryStringsToObjects(content: JSONContent[]): JSONConten
             return node
         }
 
+        let query
+
+        try {
+            query = JSON.parse(node.attrs.query)
+        } catch (e) {
+            query = DEFAULT_QUERY
+        }
```

Comment:
Swift has a nice lang. feature for such cases, one-liner and nicer to read.

`query = try? JSON.parse(node.attrs.query) ?? DEFAULT_QUERY`